MODULE.bazel

```diff
@@ -29,6 +29,10 @@ bazel_dep(
     name = "rules_license",
     version = "1.0.0",
 )
+bazel_dep(
```

Comment:
Is there no need to add anything to the WORKSPACE file?

Reply:
My impression is that anyone using `WORKSPACE` should have already been calling `rules_rust_dependencies()` from it (and indeed, our neighboring `WORKSPACE.bazel` does that) so the change in this PR to `rust/repositories.bzl` to add `rules_shell` to the list should be sufficient.

Reply:
Ah! I missed that, yeah, then it's covered. Thanks!